build.gradle

```diff
@@ -288,7 +288,7 @@ dependencies {
   integrationTestImplementation sourceSets.main.runtimeClasspath
   integrationTestImplementation sourceSets.test.runtimeClasspath
 
-  testImplementation group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.1.0'
+  testImplementation group: 'uk.gov.hmcts', name: 'befta-fw', version: '5.16.1'
```

Comment:
I don't think current version of definition-store on AAT supports NoC elements in the spreadsheet.

Reply:
We are using def store's develop branch, aren't we?

Reply:
reverted to 5.1.0

Reply:
Sorry - you were right, @smathangi . Thanks for reverting @sullivangareth .